assets/js/buttons-examples.js

```diff
@@ -145,7 +145,12 @@ $(document).ready(function () {
           classes: "add-to-cart-btn",
           text: "Add To Cart",
         },
-
+        {
+          type: "apple-store",
+          label: "Apple Store Download Button",
+          classes: "applestore",
+          text: "Download",
```

Comment:
Change the label to "App Download Button" and add a new key `variations` where you add inside it the apple store and the play store buttons. You can follow the implementation of social media buttons for this.